memtier_benchmark.cpp

```diff
@@ -151,7 +151,6 @@ static void config_print(FILE *file, struct benchmark_config *cfg)
         cfg->data_verify ? "yes" : "no",
         cfg->verify_only ? "yes" : "no",
         cfg->verify_set_only ? "yes" : "no",
-        cfg->verify_set_only ? "yes" : "no",
```

Comment:
did not you just add it couple of days ago?

Reply:
yes I asked you if you want me to just revert it from the master